roles/edpm_update/molecule/default/molecule.yml

```diff
@@ -19,6 +19,12 @@ provisioner:
   log: true
   name: ansible
   inventory:
+    host_vars:
+      test_vars:
+        working_directory: /tmp/molecule_test_dir
```

Comment:
please add a comment explaining why you think you have to use a directory to share files across stages.